pkg/kubectl/cmd/apply/apply_test.go

```diff
@@ -68,6 +69,12 @@ var (
 	codec = scheme.Codecs.LegacyCodec(scheme.Scheme.PrioritizedVersionsAllGroups()...)
 )
 
+func init() {
+	klog.InitFlags(nil)
```

Comment:
How is that related, why is it needed now?

Reply:
It doesn't relate, but after the switch from glog to klog every end user binary now needs to initialize it to set up the flags, the switch didn't fix it in tests. every _test.go is the binary in the end. You'd end up using it as `go test -race ./pkg/kubectl/cmd/get -v -run='TestWatchOnlyList' -timeout=3s -args -logtostderr=true -v=9` but without those flags registered you can't. (glog registered those flags when included in its init, klog doesn't)